source/index.js

```diff
@@ -102,31 +102,6 @@ QueryGenerator.prototype.describeTableQuery = function (...args) {
     .replace('relid', 'oid');
 };
 
-// [5] Allow BigInts on `Model.findByPk`
```

Comment:
@papb Was wondering if you had any thoughts on this since you added the patch and are more familiar with JS data types. 

Does it make sense to handle BigInts as strings?

Also @douglasselias do you know which tests / files were failing specifically because of this?

Reply:
[One example](https://github.com/cockroachdb/sequelize-cockroachdb/pull/64/checks?check_run_id=2088779688)
The error it gives is: "Invalid value 640342301583048705n" because it tries to do something like this: object[id], but id is a BigInt

Reply:
> [One example](https://github.com/cockroachdb/sequelize-cockroachdb/pull/64/checks?check_run_id=2088779688)
> The error it gives is: "Invalid value 640342301583048705n" because it tries to do something like this: object[id], but id is a BigInt

Ah I see, thanks!